### ISSUE TITLE
Double jump using trackpad

### ISSUE THREAD

**Issue by devolkpl** [alvarotrigo/multiscroll.js#60]
Hi guys,
pretty nice plugin at first.

I have problem with scrolling when I use trackpad in my macbook. I'm always getting two jumps instead just to the next one. In addition to this, it looks like jumping in random direction! Sometimes it move down, sometimes up. 
Is this a bug or I'm missing some config? 

```
            verticalCentered : false,
            scrollingSpeed: 500,
            menu: false,
            anchors: sections,
            navigation: false,
            loopBottom: false,
            loopTop: false,
            css3: true,
            paddingTop: 0,
            paddingBottom: 0,
            normalScrollElements: null,
            keyboardScrolling: true,
            touchSensitivity: 5,
            onLeave: function(index, nextIndex, direction){
                $('nav a').removeClass('active');
                $('nav li').eq(nextIndex-1).find('a').addClass('active');
            }
```

**Comment by alvarotrigo** [comment]
That's because you are scrolling very fast and the inertia continues firing the scroll event after you reach other section.

The problem was solved in its bigger brother [fullPage.js](alvarotrigo.com/fullPage/) time ago, but I never implemented the solution in multiscroll.js.

If you are really interested in having it solved, [contact me](http://alvarotrigo.com/#contact-page) for an urgent request because I'm not planning to work on it in the short term.

**Comment by devolkpl** [comment]
Ohh thanks man. Increasing scrollingSpeed helped. 
Anyway scrollingSpeed variable name is confusing :)

**Comment by csergiuu** [comment]
Hello guys!

I'm having the same issue, so we have no solution? :/

**Comment by danm0e** [comment]
Same issue with the magic mouse. Any fixes?

**Comment by alvarotrigo** [comment]
@danm0e nop, as I said:

> If you are really interested in having it solved, contact me for an urgent request because I'm not planning to work on it in the short term.

**Comment by alvarotrigo** [comment]
Fixed on the latest version multiScroll 0.2.3
